src/base/bittorrent/trackerentry.h

```diff
@@ -31,31 +31,49 @@
 
 #include <libtorrent/announce_entry.hpp>
 
+#include <QString>
 #include <QtGlobal>
 
-class QString;
+class QStringList;
+template <class T> class QVector;
```

Comment:
Why not include QContainerFwd like you've done above?

Reply:
changed